lib/orwell/github/user.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Orwell.Github.User do
```

Comment:
I tend to use `Orwell.Github` unless I'm building a module that might be extracted later into a package. Isn't `Github` too generic a name for that?

What's the common practise here?